model/src/main/kotlin/config/AnalyzerConfiguration.kt

```diff
@@ -24,13 +24,6 @@ import com.fasterxml.jackson.annotation.JsonInclude
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 data class AnalyzerConfiguration(
```

Comment:
Does Hoplite fail if the config file contains unexpected options? If so the PR needs the breaking change label.

Reply:
> Does Hoplite fail if the config file contains unexpected options?

No, it simply ignores them. We used that feature in the past to already introduce upcoming options to our config before they were implemented upstream, to have a smooth transition.